src/index.tsx

```diff
@@ -206,8 +206,8 @@ const ReactInputVerificationCode = ({
             role='button'
             tabIndex={0}
             className={`ReactInputVerificationCode__item ${
-              i === activeIndex ? 'is-active' : ''
-            }`}
+              value[i] !== placeholder ? 'is-filled' : ''
+            } ${i === activeIndex ? 'is-active' : ''} `}
```

Comment:
```suggestion
            }${i === activeIndex ? 'is-active' : ''} `}
```

Avoid extra spacing, otherwise LGTM 👏

Reply:
If I understand you correctly your suggestion may lead to className like  ->`ReactInputVerificationCode__item is-filledis-active `. We need this space.

Instead, I removed the space at the end because it was redundant :)

Reply:
You're right, my bad
Actually spaces won't break anything so let's go for it 👍